doc/nrf/ug_zigbee_qsg.rst

```diff
@@ -145,17 +145,14 @@ To program the samples, complete the following steps:
 #. Add the network coordinator application:
 
    a. In the Welcome panel or on the Welcome page, click :guilabel:`Create a new application from sample`.
```

Comment:
This is fine for now, but as a head's up we will be changing this to just say "Create a new application" either in the June or July VSC release.